src/main/webapp/css/upload.css

```diff
@@ -1,7 +1,7 @@
 /* Form is hidden by default. */
-.hidden {
+/* .hidden {
   display: none;
-}
+} */
```

Comment:
Is this no longer needed?

Reply:
Yes, now the form will be available immediately and the Blobstore upload URL will be created after it is submitted. I have removed the code.